Models/SidechainNodeModel.cs

```diff
@@ -10,5 +10,9 @@ public class SidechainNodeModel : StratisNodeModel
         public string BlockProducerHits { get; set; }
 
         public decimal BlockProducerHitsValue { get; set; }
+
+        public int BlockProducersMining { get; set; }
```

Comment:
How are these different to what is already in fields related to block producer hits

Reply:
It contains (...%) in the string as well, I'll split it off and use that instead in the model and then remove these fields.